packages/js/email-editor/src/editor.tsx

```diff
@@ -32,6 +32,11 @@ function Editor() {
 	);
 }
 
+const WrappedEditor = applyFilters(
+	'mailpoet_email_editor_the_editor_component',
```

Comment:
I think we could find a better name for the wrapping filter. What do you think?
From my experience, the determiner `the` is not usually used in filter names.
What do you think about the name `mailpoet_email_editor_wrap_editor_component `?